File: src/main/java/edu/nifu/sas/util/FileUtil.java
package edu.nifu.sas.util;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStream;
import java.io.InputStreamReader;
import java.util.UUID;

public class FileUtil {
    public static String getStringFromInputStream(InputStream inputStream) throws IOException {
        InputStreamReader inputStreamReader = new InputStreamReader(inputStream);
        BufferedReader breader=new BufferedReader(inputStreamReader);
        String buffer="";
        StringBuffer resultStr=new StringBuffer();
        while ((buffer=breader.readLine())!=null){
            resultStr.append(buffer);
        }
        return resultStr.toString();
    }

    public static String getRandomFileName(String fileName){
        int index = fileName.lastIndexOf(".");
        String suffix=fileName.substring(index+1);
        return fileName.substring(0,index)+ "-"+UUID.randomUUID()+"."+suffix;
    }


}

File: src/main/java/com/example/networkdemo/util/FileUtil.kt
package com.example.networkdemo.util

import java.io.BufferedReader
import java.io.InputStream
import java.io.InputStreamReader
import java.lang.StringBuilder

object FileUtil {
    public fun getStringFromInputStream(input:InputStream):String{
        val resultStr:StringBuilder=java.lang.StringBuilder()
        val inputStreamReader=InputStreamReader(input)
        val breader=BufferedReader(inputStreamReader)
        breader.use {
            breader.forEachLine {
                resultStr.append(it)
            }
        }
        return resultStr.toString()
    }
}